corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -172,11 +172,13 @@ def fire_change_processed_event(self, change, context):
 def handle_pillow_error(pillow, change, exception):
     from couchdbkit import ResourceNotFound
     from pillow_retry.models import PillowError
-    try:
-        meta = pillow.get_couch_db().show('domain_shows/domain_date', change['id'])
-    except ResourceNotFound:
-        # Show function does not exist
-        meta = None
+    meta = None
+    if hasattr(pillow, 'get_couch_db'):
+        try:
+            meta = pillow.get_couch_db().show('domain_shows/domain_date', change['id'])
+        except ResourceNotFound:
+            pass
```

Comment:
shouldn't this still be `meta = None`? else if the resource is not found on a couch pillow line 182 will through an error saying meta isn't defined